scripts/update_config_docs.py

```diff
@@ -28,7 +28,11 @@
 from typing import Any
 
 import yaml
-from pydantic import BaseSettings
+
+try:
+    from pydantic_settings import BaseSettings
+except ImportError:
```

Comment:
```suggestion
except ImportError:  # fallback for Pydantic v1
```

Just as a reminder why we did this. Maybe also add a comment to the deprecated `.json()` call so that we remember to clean it up later?

Reply:
Thanks for reminding me. As for the `.json()` and `.schema_json()`, we will get a deprecation warning every time we run the update_config_docs script, so we won't forget. But I added them anyway. Take a look at [9a273eb](https://github.com/ghga-de/microservice-repository-template/pull/173/commits/9a273eba4ec464133e1b891d6b8b084fb873ee16)

Reply:
Ok, thanks.